Neos.RedirectHandler/Classes/Command/RedirectionCommandController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+namespace Neos\RedirectHandler\Command;
+
+/*
+ * This file is part of the TYPO3.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use League\Csv\Reader;
+use League\Csv\Writer;
```

Comment:
"So dependency is not really important here." I can't really agree here, no matter where: dependencies are always important... In a redirect package I created last year (https://github.com/SimplyAdmire/SimplyAdmire.Redirects) we also created an import and even a writer (to write to for example an nginx config file or output for a .htaccess). I do feel like import / export / writing is a bit much for the base package though.

The one time we really used the import it was for a onetime import during a migration, and I think that's the most common usecase. That would actually be an argument for me to move such commands to a separate package (also removes the dependencies and makes the base package more slim)